redwoods-insurance-salesforce/package.json

```diff
@@ -26,7 +26,8 @@
         "eslint": "^5.15.1",
         "husky": "^3.0.5",
         "lint-staged": "^9.3.0",
-        "prettier": "^1.18.2",
+        "prettier": "1.18.2",
```

Comment:
We should allow version upgrades by prepending `^` to the version number.

Reply:
Also prettier related scripts in `package.json` don't have file extensions added for Apex classes and triggers.

Reply:
Fair, adjusting ;)